faunadb/deserialization_test.go

```diff
@@ -178,6 +178,19 @@ func TestDeserializeQueryVInsideObjectV(t *testing.T) {
 	require.Equal(t, ObjectV{"a": StringV("a"), "b": ObjectV{"lambda": QueryV{lambda}}, "c": StringV("c")}, object)
 }
 
+func TestDeserializeVersionedQueryV(t *testing.T) {
+	var query QueryV
+
+	lambda1 := json.RawMessage(`{"lambda": "x", "api_version": "3", "expr": {"var": "x"}}`)
+	lambda2 := json.RawMessage(`{"lambda": "x", "api_version": "2.12", "expr": {"var": "x"}}`)
+
+	require.NoError(t, decodeJSON(`{"@query": {"lambda": "x", "api_version": "3", "expr": {"var": "x"}}}`, &query))
+	require.Equal(t, QueryV{lambda1}, query)
+
+	require.NoError(t, decodeJSON(`{"@query": {"lambda": "x", "api_version": "2.12", "expr": {"var": "x"}}}`, &query))
```

Comment:
remove `"api_version": "2.12"` from this json, the rest should remain as is

Reply:
I've removed this entirely although I dont understand the reasoning behind that since its a deserialization test

Reply:
sorry, I was under the impression this was doing a call to the server before deserialize, feel free to restore the old commit